memory-bank/projectbrief.md

```
@@ -23,20 +23,20 @@ a16n is a CLI tool and library that translates agent customization between codin
 - **Language**: TypeScript
 - **Package Manager**: pnpm (workspaces)
 - **Build Orchestration**: Turborepo
-- **Versioning**: Changesets
-- **Testing**: Vitest (planned)
+- **Versioning**: Release-Please
+- **Testing**: Vitest
+- **Documentation**: Docusaurus
 
-## Project Structure (Planned)
+## Project Structure
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)